pkg/sanitization/aws/s3/s3.go

```diff
@@ -1,4 +1,4 @@
-package resources
+package s3
```

Comment:
How come `s3` is its own sanitization package and `iam`/`lambda`/etc are together at top-level?

Reply:
need to refactor this, i didnt mean to touch it let me change it back